File: 6/6.rs
fn main() {
    let sum_of_squares: u64 = (1..101).map(|x: u64| x.pow(2)).sum();
    let square_of_sums: u64 = (1..101).sum::<u64>().pow(2);
    println!("{:#?}", square_of_sums - sum_of_squares);
}

File: 6/6.go
package main

import (
	"fmt"
)

func sumOfSquares(n int) int {
	sum := 0
	for i := 1; i <= n; i++ {
		sum += i * i
	}
	return sum
}

func squareOfSums(n int) int {
	sum := 0
	for i := 1; i <= n; i++ {
		sum += i
	}
	return sum * sum
}

func main() {
	fmt.Println(squareOfSums(100) - sumOfSquares(100))
}
